### PR TITLE
libcni: find plugin in exec

### PR DESCRIPTION
**Pull request by mars1024**
Function `validatePlugin` in libcni should find plugin by `exec.FindInPath` instead of using `invoke.FindInPath` directly, just like `addNetwork` [https://github.com/containernetworking/cni/blob/df124b22e4f7a0b716d313884aee6474b319b4a3/libcni/api.go#L378](https://github.com/containernetworking/cni/blob/df124b22e4f7a0b716d313884aee6474b319b4a3/libcni/api.go#L378).

I think this breaks CI of [https://github.com/cri-o/ocicni/pull/51](https://github.com/cri-o/ocicni/pull/51).

### CODE REVIEW

File: libcni/api.go [hunk 1]
```diff
@@ -580,7 +580,7 @@ func (c *CNIConfig) ValidateNetworkList(ctx context.Context, list *NetworkConfig
 // It uses the same logic as ValidateNetworkList)
 // Returns a list of capabilities
 func (c *CNIConfig) ValidateNetwork(ctx context.Context, net *NetworkConfig) ([]string, error) {
-	caps := []string{}
+	var caps []string
```
Comment (dcbw) [inline libcni/api.go]: @mars1024 seems like an unrelated change?

Comment (mars1024) [reply libcni/api.go]: yeap, maybe some auto typo fix ~ I will revert it.

Comment (mars1024) [reply libcni/api.go]: Updated, thanks @dcbw
